packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -255,17 +266,33 @@ function TokenRow({ style, onClick, token }: TokenRowProps): JSX.Element {
             <div className="flex flex-col items-start space-y-1">
               {/* TODO: anchor shouldn't be nested within a button */}
               {isDepositMode ? (
-                <a
-                  href={`${getExplorerUrl(l1Network.id)}/token/${
-                    token.address
-                  }`}
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  className="text-xs text-blue-link underline"
-                  onClick={e => e.stopPropagation()}
-                >
-                  {shortenAddress(token.address).toLowerCase()}
-                </a>
+                <>
+                  {isL2NativeToken ? (
```

Comment:
Not a blocker but we should try and reduce the ternary nested JSXs as most of them have same code with minor variations - maybe extract `tokenLink (href)` and `tokenAddress` out in the component and use them here.